style.css

```diff
@@ -0,0 +1,110 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap');
+@import url('https://fonts.cdnfonts.com/css/poppins');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-repeat: no-repeat;
+  font-family: inter, sans-serif;
+}
+
+.topnav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  min-height: 48px;
+  background-color: #fff;
+  margin: 0 25px;
+}
+
+.logo {
+  font-family: poppins, sans-serif;
+  font-style: bold;
+  font-size: 18px;
+  line-height: 20px;
+  line-height: 74%;
+  font-weight: 700;
+  color: #6070ff;
+  text-decoration: none;
+}
+
+.hambuger {
+  display: block;
+}
+
+.hambuger-bar {
+  width: 26px;
+  height: 3px;
+  background-color: #6070ff;
+  margin: 5px;
+}
+
+.container {
+  background-image: url("./images/header-shapes\ mobile.png");
```

Comment:
- **[OPTIONAL]** The background image for the headline section looks amazing at some screen sizes 👏🏼 , however, it starts crashing at some others a proper fix for this is from the Figma design system export each piece highlighted in red square in the following screenshot and set them using CSS multiple background image 👍🏼 with setting the right position using a `%` unit for each asset.

### Your Design
![image](https://user-images.githubusercontent.com/68030297/229313239-c2bf8e78-97a8-4047-a63c-a062e77fd569.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/229313350-6eb9da3c-d76f-458c-8138-5c43e17117b6.png)

For more information visit [CSS Multiple Backgrounds](https://www.w3schools.com/css/css3_backgrounds.asp)